content/en/tracing/service_catalog/integrations.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Using Integrations with Service Catalog
+kind: documentation
+further_reading:
+- link: "/tracing/service_catalog/service_definition_api/"
+  tag: "Documentation"
+  text: "Service Definition API"
+---
+
+## PagerDuty integration
+You can add PagerDuty metadata to a service so that Service Catalog it displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service.
```

Comment:
```suggestion
You can add PagerDuty metadata to a service so that Service Catalog displays and links to information such as who is on-call and whether there are active PagerDuty incidents for the service.
```